<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-9B42UR3" failures="4" tests="10" name="Test" time="27.383" errors="0" timestamp="2025-08-10T11:45:12 IST">
  <testcase classname="api.test.DDTests" name="testPostuser" time="10.271"/>
  <testcase classname="api.test.DDTests" name="testPostuser" time="1.148"/>
  <testcase classname="api.test.DDTests" name="testPostuser" time="1.25"/>
  <testcase classname="api.test.DDTests" name="testPostuser" time="1.511"/>
  <testcase classname="api.test.DDTests" name="testPostuser" time="1.341"/>
  <testcase classname="api.test.DDTests" name="testDeleteuser" time="1.344">
    <failure type="java.lang.AssertionError" message="Validatating response code :Uname012 expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: Validatating response code :Uname012 expected [200] but found [404]
at api.test.DDTests.testDeleteuser(DDTests.java:109)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteuser -->
  <testcase classname="api.test.DDTests" name="testDeleteuser" time="1.106">
    <failure type="java.lang.AssertionError" message="Validatating response code :Uname013 expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: Validatating response code :Uname013 expected [200] but found [404]
at api.test.DDTests.testDeleteuser(DDTests.java:109)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteuser -->
  <testcase classname="api.test.DDTests" name="testDeleteuser" time="1.222"/>
  <testcase classname="api.test.DDTests" name="testDeleteuser" time="1.153">
    <failure type="java.lang.AssertionError" message="Validatating response code :Uname015 expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: Validatating response code :Uname015 expected [200] but found [404]
at api.test.DDTests.testDeleteuser(DDTests.java:109)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteuser -->
  <testcase classname="api.test.DDTests" name="testDeleteuser" time="1.134">
    <failure type="java.lang.AssertionError" message="Validatating response code :Uname016 expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: Validatating response code :Uname016 expected [200] but found [404]
at api.test.DDTests.testDeleteuser(DDTests.java:109)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteuser -->
</testsuite> <!-- Test -->
